budgetportal/models.py

```diff
@@ -829,6 +1152,7 @@ def __init__(self, **kwargs):
         self.category = kwargs['category']
         self._openspending_api = None
         self.package = kwargs['package']
+        self._adjusted_budget_summary = {}
```

Comment:
I think this was added by mistake. This caching on `Department` is also me being silly and we can clean that up.

Reply:
yep, shouldn't be here - thanks for spotting that.